### ISSUE TITLE
Add possibility to define custom headers response

### ISSUE THREAD

**Issue by nexxus-vi** [Apipie/apipie-rails#706]
Hi, i'm using [pagy](https://github.com/ddnexus/pagy) in an API Only project to paginate model's index. This gem provides additional header in response, such as 

```
Link - <http://localhost:3000/api/v1/accounts?page=1>; rel="first", <http://localhost:3000/api/v1/accounts?page=1>; rel="last"

Current-Page - 1

Page-Items - 2
```

Now i can add headers for request like this: 

```
header 'Authorization', 'Bearer [your_access_token]', required: true
header 'Content-Type', 'application/json', required: true
header 'Accept', 'application/json', required: true

```

but how about headers for response?

**Comment by PanosCodes** [comment]
Sure

What to you think about https://github.com/Apipie/apipie-rails/pull/924

**Comment by mathieujobin** [comment]
released in v1.4.0
